source/isaaclab/isaaclab/sim/spawners/from_files/from_files.py

```diff
@@ -182,10 +196,86 @@ def spawn_ground_plane(
 """
 
 
+def _create_prim(
```

Comment:
Shall we make an MR of this in IsaacSim and fix it over there? We can avoid duplicating their function here then.